src/app/view/components/dot-navigation/components/dot-sub-nav/dot-sub-nav.component.ts

```diff
@@ -8,7 +8,7 @@ import { DotMenu, DotMenuItem } from '@models/navigation';
             state(
                 'expanded',
                 style({
-                    height: '*',
+                    height: '!',
```

Comment:
There was a little jump when the accordion animates, this fix it. More info: https://github.com/angular/angular/issues/15507#issuecomment-305411263